airQuality/jsonld-contexts/airQuality.jsonld

```diff
@@ -25,6 +25,7 @@
       "no": "https://vocab.egm.io/no",
       "no2": "https://vocab.egm.io/no2",
       "o3": "https://vocab.egm.io/o3",
+      "occupancySchedule": "https://vocab.egm.io/occupancySchedule",
```

Comment:
not very fan of this in air quality context, I guess we don't have better options